### PR TITLE
Add honggfuzz to project setup.

### PR DESCRIPTION
**Pull request by oliverchang**
#1128.

### CODE REVIEW

File: src/appengine/handlers/cron/project_setup.py [hunk 1]
```diff
@@ -119,6 +119,12 @@ def job_name(self, project_name):
     minimize_job_override=LIBFUZZER_ASAN_JOB)
 NO_ENGINE_ASAN_JOB = JobInfo('asan_', 'none', 'address', [])
 
+HONGGFUZZ_ASAN_JOB = JobInfo(
+    'honggfuzz_asan_',
+    'honggfuzz',
+    'address', ['honggfuzz', 'engine_asan'],
+    minimize_job_override=LIBFUZZER_ASAN_JOB)
```
Comment (inferno-chromium) [inline src/appengine/handlers/cron/project_setup.py]: Eventually we want to support native minimization using fuzzing engine itself, does honggfuzz support that ?

Comment (oliverchang) [reply src/appengine/handlers/cron/project_setup.py]: I don't think so.. (https://github.com/google/honggfuzz/blob/master/docs/USAGE.md)

**Comment by oliverchang** [comment]
> LGTM.
> 
> I think we need to update environment.is_engine_fuzzer_job and any other similar places that check against libfuzzer, afl. like search for is_afl_job, is_libfuzzer_job

Done in https://github.com/google/clusterfuzz/pull/1197.

**Pull request merged**